datadog_checks_base/tests/test_http.py

```diff
@@ -414,6 +415,52 @@ def test_unknown_name_default(self):
         assert http.options['verify'] is True
 
 
+class TestLogger:
+    def test_default(self, caplog):
+        check = AgentCheck('test', {}, [{}])
+
+        with mock.patch('requests.get'):
+            check.http.get('https://www.google.com')
+
+        expected_message = 'Sending GET request to https://www.google.com'
+        for _, _, message in caplog.record_tuples:
+            assert message != expected_message
+
+    def test_instance(self, caplog):
+        instance = {'log_requests': True}
+        init_config = {}
```

Comment:
Should we also test the case when`init_config['log_requests']` is false but `instances['log_requests']` is true to make sure the init_config value gets overriden?